### ISSUE TITLE
How to do that with multi-processing

### ISSUE THREAD

**Issue by amazingProj** [lemmingapex/trilateration#33]
Hii 
I try to understand how to do it parallel, 
do you have any idea?

**Comment by kburfi** [comment]
The trilateration itself is done by the Levenberg–Marquardt algorithm, which is a standard implementation. So calculating the result for a single problem in parallel is not simple and would lead you to implementing a new algorithm. Hence this is way out of scope.
You could of course just run the trilateration with multiple positions in parallel. E. g. using different threads or parllel streams..
